src/app/public/modules/tabs/tab-button.component.html

```diff
@@ -1,6 +1,5 @@
 <a
     class="sky-btn-tab sky-margin-inline-compact"
-    preserveFragment
```

Comment:
This shouldn't have been set in the first place since we want fragments to be reset when tabs are clicked.